src/wdio/services/TerraService.js

```diff
@@ -69,6 +73,8 @@ export default class TerraService {
   // eslint-disable-next-line class-methods-use-this
   afterCommand(commandName) {
     if (commandName === 'refresh' || (commandName === 'url')) {
+      hideInputCaret('body');
```

Comment:
Why are we adding this?

Reply:
https://github.com/cerner/terra-toolkit/pull/312#discussion_r304207761

Reply:
@mjhenkes  😄 saw that later one. Still think a comment will be helpful for later

Reply:
I'd put this inside the try.

Reply:
I agree. Then if something weird does occur, it will be hidden from consumers since we are trying just trying to help 😄

Reply:
Done in 368a5aa21c202fa551437dd994ee11a4d65bb46c